File: src/codes/Users_Manual/Tutorial/intensity/focus.py
~from pyautd3 import Intensity, Focus, FocusOption
~x = 1.0
~y = 0.0
~z = 0.0
Focus(
    pos=[x, y, z],
    option=FocusOption(
        intensity=Intensity(0x80),
    ),
)

File: src/codes/Users_Manual/Tutorial/intensity/focus.rs
# use autd3::prelude::*;
# fn main() {
# let x = 0.0;
# let y = 0.0;
# let z = 0.0;
# let _ = 
Focus {
    pos: Point3::new(x, y, z),
    option: FocusOption {
        intensity: Intensity(0x80),
        ..Default::default()
    },
};
# }